src/SmartComponents/BaselinesPage/BaselinesKebab/redux/helpers.js

```diff
@@ -0,0 +1,52 @@
+function convertFactsToCSV(data) {
+    if (data === null || !data.length) {
+        return null;
+    }
+
+    let columnDelimiter = data.columnDelimiter || ',';
+    let lineDelimiter = data.lineDelimiter || '\n';
+
+    let headers = 'Name,Last Sync,';
+    let result = headers + lineDelimiter;
+
+    data.forEach(function(baseline) {
+        baseline.forEach(function(detail) {
+            result += detail;
+            result += columnDelimiter;
+        });
+
+        result += lineDelimiter;
+    });
+
+    return result;
+}
+
+function downloadCSV(baselinesTableData) {
+    let csv = convertFactsToCSV(baselinesTableData);
+
+    if (csv === null) {
+        return;
+    }
+
+    let filename = 'baseline-list-export-';
+    let today = new Date();
+    filename += today.getFullYear() + '-' + (today.getMonth() + 1) + '-' + today.getDate();
+    filename += '_';
+    filename += today.getHours() + ':' + today.getMinutes() + ':' + today.getSeconds();
+    filename += '.csv';
```

Comment:
it doesnt block merging of this PR, but is there a way to just get the datetime as an ISO8601 string and use that?